src/mjolnir/graphbuilder.cc

```diff
@@ -983,6 +985,11 @@ void BuildTileSet(const std::string& ways_file,
         // Set drive on right flag
         graphtile.nodes().back().set_drive_on_right(dor);
 
+        // set the osmid for the node
+        if (include_osmids) {
+          graphtile.set_last_node_osmid(node.osmid_);
```

Comment:
maybe its confusing, but the very act of adding ids to the vector in the tile builder object should be enough to signal that we are including node ids. then we dont need that boolean flag in the constructor. what do you think?

Reply:
Done.